packages/babel-preset-flow/src/index.js

```diff
@@ -0,0 +1,9 @@
+import transformSyntaxFlow from "babel-plugin-syntax-flow";
+import transformFlowStripTypes from "babel-plugin-transform-flow-strip-types";
+
+export default {
+  plugins: [
+    transformSyntaxFlow,
```

Comment:
I see this was in there, weird. This isn't needed, the transform already loads this: https://github.com/babel/babel/blob/master/packages/babel-plugin-transform-flow-strip-types/src/index.js#L5